ee/control/control.go

```diff
@@ -79,7 +79,7 @@ func (cs *ControlService) Start(ctx context.Context) {
 	for {
 		// Fetch immediately on each iteration, avoiding the initial ticker delay
 		if err := cs.Fetch(); err != nil {
-			level.Debug(cs.logger).Log(
```

Comment:
I'm fine with anything above debug -- I just wanted them to show up in non-debug mode.

Reply:
Oh okay, I don't think I realized the distinction -- I've been tailing debug.json while testing instead of the system logs. I can move these back to debug, then.

Reply:
These are back to debug and I updated the ones in notify_consumer as well 👍

Reply:
I added a potential project of making this all simpler.